style.css

```diff
@@ -0,0 +1,331 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+@import url('https://fonts.cdnfonts.com/css/cocogoose');
+
+* {
+  font-family: 'lato', sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  scroll-behavior: smooth;
+}
```

Comment:
- [ ] Please make sure that your website pages look almost identical to the original design and use the exact same colors, for text, backgrounds, borders, etc, and have the following colors #272A31, #EC5242, #D3D3D3. :+1: